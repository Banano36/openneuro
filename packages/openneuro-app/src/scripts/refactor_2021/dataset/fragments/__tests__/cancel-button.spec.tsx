import React from 'react'
import { render } from '@testing-library/react'
import { CancelButton } from '../cancel-button'

describe('CancelButton component', () => {
  it('renders with default props', () => {
    const { asFragment } = render(<CancelButton action={() => {}} />)
    expect(asFragment()).toMatchSnapshot()
  })
})
