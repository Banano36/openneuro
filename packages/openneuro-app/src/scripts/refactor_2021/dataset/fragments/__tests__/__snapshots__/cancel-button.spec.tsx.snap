// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CancelButton component renders with default props 1`] = `
<Button
  className="description-btn description-button-cancel"
  icon="fas fa-times"
  label="Cancel"
  onClick={[Function]}
/>
`;
